### DISCUSSION TITLE
VSCode jupyter extension (on Insider) appears to take a long time to get saved after being used for a while. How to mitigate?

### DISCUSSION THREAD

**Discussion by shandou** [microsoft/vscode-jupyter#15365] [Questions and Answers] [answered]
In the past 1-2 months or so, I start to observe that jupyter notebooks in VSCode could run normally for a while, and then start to take a very long time to run a cell or get saved. This can be resolved by running "reload window", but notebook would also lose all the states. Has anyone run into similar problems and are there better ways to mitigate? Thank you!

The screen capture below is an example where this status box would display "saving..." for an unusually long time.

![image](https://github.com/microsoft/vscode-jupyter/assets/15528558/fdcd658d-37a7-4be4-9021-e699cd6e7f5c)

**Comment by DonJayamanne** [comment, marked as answer]
Closing in favour of https://github.com/microsoft/vscode-jupyter/issues/15366
